pom.xml

```diff
@@ -4,14 +4,18 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532.3</version>
+    <version>2.31-SNAPSHOT</version>
   </parent>
   <artifactId>validating-string-parameter</artifactId>
   <packaging>hpi</packaging>
   <name>Validating String Parameter Plugin</name>
-  <version>2.4-SNAPSHOT</version>
+  <version>2.5-SNAPSHOT</version>

```

Comment:
Is there a reason the version needed to be bumped here? Usually the pom is left in a place that is correct for the next round of development.